src/us/kbase/auth2/providers/OrcIDIdentityProviderFactory.java

```diff
@@ -97,8 +97,14 @@ public Set<String> getEnvironments() {
 
 		// state will be url encoded
 		@Override
-		public URI getLoginURI(final String state, final boolean link, final String environment)
+		public URI getLoginURI(
+				final String state,
+				final String pkceCodeChallenge,
+				final boolean link,
+				final String environment)
 				throws NoSuchEnvironmentException {
+			// note that OrcID does not currently implement PKCE so we ignore the code
```

Comment:
The standard rendering is `ORCID` (all caps), although `OrcID` is definitely more amusing.

Reply:
That's why I do it :P